tests/test_preload_script.py

```diff
@@ -602,6 +602,68 @@ async def test_preloadScript_add_runImmediately(websocket, html, context_id,
         assert result["result"] == {"type": "undefined"}
 
 
+@pytest.mark.asyncio
+async def test_preloadScript_addGlobally_withBlankTargetLink(
+        websocket, context_id, html, read_sorted_messages, get_cdp_session_id):
+    LINK_WITH_BLANK_TARGET = html(
+        '<a href="https://example.com" target="_blank">new tab</a>')
+
+    await execute_command(
+        websocket, {
+            "method": "script.addPreloadScript",
+            "params": {
+                "functionDeclaration": """() => {
+                    console.log('my preload script', window.location.href);
+                }""",
+            }
+        })
+
+    await execute_command(
+        websocket, {
+            "method": "browsingContext.navigate",
+            "params": {
+                "context": context_id,
+                "url": LINK_WITH_BLANK_TARGET,
+                "wait": "complete",
+            }
+        })
+
+    await subscribe(websocket, ["log.entryAdded"])
+
+    session_id = await get_cdp_session_id(context_id)
+
+    command_id = await send_JSON_command(
+        websocket, {
+            "method": "cdp.sendCommand",
```

Comment:
why do we send a CDP command instead of a BiDi command? Is it for the gesture flag? perhaps a todo pointing to https://github.com/w3c/webdriver-bidi/issues/359 would be worthwhile

Reply:
Yes! Will do